src/wasm-binary.h

```diff
@@ -906,8 +906,13 @@ class WasmBinaryBuilder {
   BinaryConsts::ASTNodes readExpression(Expression*& curr);
   void pushBlockElements(Block* curr, size_t start, size_t end);
   void visitBlock(Block *curr);
-  Expression* getMaybeBlock(WasmType type);
+
+  // Gets a potential list of instructions. This is not a block and cannot be
+  // branched to.
+  Expression* getList(WasmType type);
+  // Gets a potential block. This may be branched to.
```

Comment:
I don't really understand what you mean by "potential" here. This always returns an actual block, no? Is the difference just that it may or may not be an implicit block (e.g. a function body or if arm?

Reply:
Ok, I see that's wrong. So the only difference from `getList` is that this could return a single instruction or an explicit block (that can be branched to).

Reply:
Yeah. Both can return a block or a single instruction if a block isn't needed. A block might be needed if we branch to it (in getBlock, but not getList where we assume no branches), or if we have more than one instruction.